rpc_jobs/multi_node_aio.yml

```diff
@@ -123,35 +123,24 @@
               Destroy Slave
 
     dsl: |
-      node(){{
+      common.shared_slave(){{
         try {{
-          dir("rpc-gating"){{
-            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
-            pubCloudSlave = load 'pipeline_steps/pubcloud.groovy'
-            ssh_slave = load 'pipeline_steps/ssh_slave.groovy'
-            common = load 'pipeline_steps/common.groovy'
-            multi_node_aio_prepare = load 'pipeline_steps/multi_node_aio_prepare.groovy'
-            deploy = load 'pipeline_steps/deploy.groovy'
-            maas = load 'pipeline_steps/maas.groovy'
-            tempest = load 'pipeline_steps/tempest.groovy'
-            horizon = load 'pipeline_steps/horizon.groovy'
-            kibana = load 'pipeline_steps/kibana.groovy'
-            holland = load 'pipeline_steps/holland.groovy'
-            github = load 'pipeline_steps/github.groovy'
-          }}
           instance_name = common.gen_instance_name()
           deploy_node = null
           pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
```

Comment:
Also needs to be changed for the deletion, but I can't comment down there!
`pubCloudSlave.delPubCloudSlave(instance_name: instance_name)`